report/report.go

```diff
@@ -111,3 +110,72 @@ func (r Report) LocalNetworks() []*net.IPNet {
 	}
 	return ipNets
 }
+
+// OriginTable produces a table (to be consumed directly by the UI) based on
+// an origin ID, which is (optimistically) a node ID in one of our topologies.
+func (r Report) OriginTable(originID string) (Table, bool) {
```

Comment:
I guess I can do it, but why is a free function that takes a report as the first parameter better than a method on report?